File: Clash of Code/Compare the Squares/Compare the squares.cs
using System;

class Solution
{
    static void Main(string[] args)
    {
        int N = int.Parse(Console.ReadLine());
        
        for (int i = 0; i < N; i++)
	{
		string[] input = Console.ReadLine().split(" ");
		double a = int.parse(input[0]);
        	double b = int.parse(input[1]);

        	double f = pow(a,2) + pow(b,2);
        	double s = pow(a+b,2);
	        if(f>s)
        	    Console.WriteLine(">");
	        else if(f==s)
        	    Console.WriteLine("=");
        	else
            	    Console.WriteLine("<");
        }
    }
}

File: Clash of Code/Compare the Squares/Compare the squares.py
import sys
import math

n = int(input())
for i in range(n):
    a, b = [int(j) for j in input().split()]

    if a**2+b**2>(a+b)**2:
        print(">")
    elif a**2+b**2==(a+b)**2:
        print("=")
    else:
        print("<")